ui/ceval/src/view.ts

```diff
@@ -18,7 +18,16 @@ const gaugeTicks: VNode[] = [...Array(8).keys()].map(i =>
 function localEvalInfo(ctrl: ParentCtrl, evs: NodeEvals): Array<VNode | string> {
   const ceval = ctrl.getCeval(),
     trans = ctrl.trans;
+  console.log('checking if analysable2');
   if (!evs.client) {
+    if(!ceval.analyzable) {
+      console.log('checking if analysable');
+      console.log('checking if analysable');
+      
+      return [
+        trans.noarg('Engine cannot analyze this position'),
```

Comment:
This is not how translations work. You might want to give [How translations work](https://github.com/ornicar/lila/wiki/How-translations-work) a read. But it's probably fine to just leave this untranslated anyway, so just remove the `trans.noarg` and return the string directly.